### ISSUE TITLE
paymentToken send to Mollie is different than the paymentToken returned by placeOrder

### ISSUE THREAD

**Issue by paales** [mollie/magento2#430]
**Describe the bug**

The result of placeOrder gives back a mollie_payment_token which is stored in Magento's DB.

```graphql
mutation MolliePlaceOrder($cartId: String!) {
  placeOrder(input: { cart_id: $cartId }) {
    order {
      order_number
      mollie_payment_token
      mollie_redirect_url
    }
  }
}
```

The data send to mollie is different however.

mollieProcessTransaction will fail however because the paymentToken is different in the URL than what is present in the database..

```
mutation MollieProcessTransaction($paymentToken: String!) {
  mollieProcessTransaction(input: { payment_token: $paymentToken }) {
    cart {
      id
    }
    paymentStatus
  }
}
```

**Used versions**
- *Magento Version number(eg 2.3.5)*: 2.4.2
- *Open source/Enterprise/B2b*: Open source
- *Mollie version number (Check configuration)*:  1.29

**To Reproduce**
1. Configure the custom return URL to include the {{payment_token}}
2. Create a GraphQL Cart
3. Place the order via GraphQL
4. Log the `mollie_payment_token`
5. Complete the payment on the mollie website by following the mollie_redirect_url
6. Now when redirected the {{payment_token}} in the URL is different that the mollie_payment_token.

**Expected behavior**
The payment token is the same in mollie as in the magento db

**Actual behavior**
Tokens aren't the same

**Screenshots**

<img width="1154" alt="Schermafbeelding 2021-09-23 om 13 51 33" src="https://user-images.githubusercontent.com/1244416/134502113-edd6b4bb-0c5d-468b-8aec-bb1150261bd8.png">
<img width="1137" alt="Schermafbeelding 2021-09-23 om 13 42 54" src="https://user-images.githubusercontent.com/1244416/134501112-b011a16d-1ea1-4618-8f97-a893f14f120d.png">

**Comment by Frank-Magmodules** [comment]
Hi @paales 

We are happy to share that we've just released the all-new [2.0.0](https://github.com/mollie/magento2/releases/tag/v2.0.0) version with a fix for this issue. Thank you for the detailed issue report. We are closing this issue now but please feel free to reopen the issue if this still occurs.

**Comment by Frank-Magmodules** [comment]
HI @paales ,

Yes, sorry for that, seems for some reason the solution didn't come through in the right way, sorry for that. Will make sure it's there in the upcoming release! Thank you for your patience!
